### PR TITLE
Add documentation to generate function.zip for aws-custom-runtime, graalvm and gradle

### LINKED ISSUE #723: doc Gradle buildNativeLambda  in Readme for aws-lambda and graalvm
For applications or serverless functions with aws-lambda and graalvm instructions about how to generate the ZIP file to be deployed should be included

We should tell the user to run 

`./gradlew buildNativeLambda` to generate the zip

### PR DESCRIPTION
**Pull request by ilopmar**
Fixes #723

### CODE REVIEW

File: starter-core/src/main/java/io/micronaut/starter/feature/awslambdacustomruntime/templates/awsCustomRuntimeReadme.rocker.raw [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Deployment with GraalVM
+
+If you want to deploy to AWS Lambda as a GraalVM native image, run:
+
+```bash
+./gradlew buildNativeLambda -Pmicronaut.runtime=lambda
```
Comment (sdelamo) [inline starter-core/src/main/java/io/micronaut/starter/feature/awslambdacustomruntime/templates/awsCustomRuntimeReadme.rocker.raw:6]: @ilopmar `-Pmicronaut.runtime=lambda` should not be necessary right?

File: starter-core/src/test/groovy/io/micronaut/starter/feature/awslambdacustomruntime/AwsLambdaCustomRuntimeSpec.groovy [hunk 2]
```diff
@@ -29,6 +30,17 @@ class AwsLambdaCustomRuntimeSpec extends ApplicationContextSpec  implements Comm
...
 
+    @Issue("https://github.com/micronaut-projects/micronaut-starter/issues/723")
+    void 'test readme.md with feature aws-lambda-custom-runtime and graalvm contains extra documentation'() {
+        when:
+        def output = generate(['aws-lambda-custom-runtime', 'graalvm'])
+        def readme = output["README.md"]
+
+        then:
+        readme
+        readme.contains("./gradlew buildNativeLambda -Pmicronaut.runtime=lambda")
```
Comment (sdelamo) [inline starter-core/src/test/groovy/io/micronaut/starter/feature/awslambdacustomruntime/AwsLambdaCustomRuntimeSpec.groovy:46]: `-Pmicronaut.runtime=lambda` should not be necessary right

File: starter-core/src/main/java/io/micronaut/starter/feature/awslambdacustomruntime/templates/awsCustomRuntimeReadme.rocker.raw (hunk 1 above)
Comment (ilopmar) [reply starter-core/src/main/java/io/micronaut/starter/feature/awslambdacustomruntime/templates/awsCustomRuntimeReadme.rocker.raw:6]: It is necessary if you create an application and by default you have `runtime("netty")` so you can start your application normally but then want to deploy it to AWS Lambda + GraalVM

File: starter-core/src/test/groovy/io/micronaut/starter/feature/awslambdacustomruntime/AwsLambdaCustomRuntimeSpec.groovy (hunk 2 above)
Comment (ilopmar) [reply starter-core/src/test/groovy/io/micronaut/starter/feature/awslambdacustomruntime/AwsLambdaCustomRuntimeSpec.groovy:46]: Same as before

**Review by sdelamo — APPROVED** [review]

**Pull request merged**